lib/usdUfe/ufe/UsdUndoableClearSceneItemMetadataCommand.h

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#pragma once
+
+#include <mayaUsd/base/api.h>
```

Comment:
Should be <usdUfe> - you cannot include mayaUsd here.

Reply:
I replaced that in both files with:

`#include <usdUfe/base/api.h>`